static/js/message_viewport.js

```diff
@@ -48,19 +48,24 @@ export function message_viewport_info() {
     // Return a structure that tells us details of the viewport
     // accounting for fixed elements like the top navbar.
     //
-    // The message_header is NOT considered to be part of the visible
+    // Sticky message_header is considered to be part of the visible
```

Comment:
The inversion of this comment is wrong. You’re still adding the sticky header height to `visible_top`, which is subtracted out of `visible_height`, so it’s not considered part of the visible message pane.

Reply:
Yeah, thanks for noticing.